rs/nns/governance/canister/governance.did

```diff
@@ -809,6 +931,22 @@ type ProposalData = record {
   wait_for_quiet_state : opt WaitForQuietState;
   executed_timestamp_seconds : nat64;
   original_total_community_fund_maturity_e8s_equivalent : opt nat64;
+
+  // The total of potential voting power of neurons when this proposal was
+  // created.
+  //
+  // This is the baseline that determines the amount of voting rewards (in the
+  // form of additional maturity) that voting neurons receive.
+  //
+  // For example, suppose neuron N has 42 voting power in ballot B for this
+  // proposal. The amount of voting power in B is determined by the neuron's
+  // deciding voting power when the proposal was created. If this field has 99,
+  // then, neuron N receives 42 / 99 of the rewards purse for this proposal.
+  //
+  // Note that this is NOT used to determine whether the proposal is adopted or
+  // rejected. Instead, the total subfield in the latest_tally field is used as
+  // the baseline when deciding proposals.
```

Comment:
I suggest also clarifying whether `total_potential_voting_power` is used in early adoption. The total subfield in the latest_tally field seems to be about the path where the proposal reaches its deadline without early adoption.